.github/workflows/backend.yml

```diff
@@ -1,10 +1,18 @@
 # This is a basic workflow to help you get started with Actions
 
-name: CI
+name: BackendCI
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
-on: [push, pull_request]
+on: 
+  push:
+    branches: [master]
```

Comment:
Can we remove branches:[master] in both cases? I think there's value in running CI on push to any branch.
Also, we can't push to master directly.

Reply:
it is now, forgot to add that line

Reply:
true, but should we have the workflow run on every push, even when we dont have a PR up?

Reply:
It's free, so I don't see any reasons not to run everything. It would save us from opening PRs from branches with failed builds.
If anything, I would be more inclined on removing checks on PRs because they are necessarily preceded by a push.

Reply:
fair point, I'll switch over to putting htem on just pushes